.github/workflows/ci.yml

```diff
@@ -36,12 +36,13 @@ jobs:
         timeout-minutes: 10
 
         strategy:
+            fail-fast: false
             matrix:
                 versions: [
                     {'python': '3.8', 'aiida-core': '1.6'},
                     {'python': '3.8', 'aiida-core': '2.0'},
                     {'python': '3.9', 'aiida-core': '2.0'},
-                    {'python': '3.10', 'aiida-core': '2.0'},
+                    {'python': '3.10', 'aiida-core': '2.2.1'},
```

Comment:
This is not necessary. On line 76 I install with the `~=` operator, which, when combined with `2.0`, will automatically select the latest version of the v2 major version. If you specify `2.2.1` however, it will actually pin it to `2.2` and install the latest patch. So best to just leave it to `2.0` and it will always install the latest version.

Reply:
Oh, that's a very good point, I completely missed that, thanks!